operators/src/engine/execution_context.rs

```diff
@@ -56,9 +56,10 @@ pub trait ExecutionContext: Send
         path: WorkflowOperatorPath,
     ) -> Box<dyn InitializedPlotOperator>;
 
-    async fn read_ml_model(&self, path: PathBuf) -> Result<String>;
+    async fn load_ml_model(&self, model_id: ModelId) -> Result<String>;
 
-    async fn write_ml_model(&mut self, path: PathBuf, ml_model_str: String) -> Result<()>;
+    async fn store_ml_model_in_db(&mut self, model_id: ModelId, ml_model_str: String)
```

Comment:
do we really need this? Operators do not produce new models.

Reply:
ba20a942

---

Commit MSG:

refactor: remove store_ml_model methods from execution context